docs/version_db_auth.md

```diff
@@ -8,21 +8,34 @@
 |--------------------|--------------|------------|-----|------|---------|-------|---------|
 | [sql_rev][1]       | VARCHAR(100) |            | PRI | NO   |         |       |         |
 | [required_rev][2]  | VARCHAR(100) |            | MUL | YES  |         |       |         |
-| [2020_02_07_00][3] | bit(1)       |            |     | YES  |         |       |         |
+| [date][3]          | VARCHAR(50)  |            |     | YES  |         |       |         |
+| [2021_06_17_00][4] | BIT(1)       |            |     | YES  |         |       |         |
 
 [1]: #sql_rev
 [2]: #required_rev
 [3]: #2020_02_07_00
+[4]: #2021_06_17_00
+
+**Description of the fields**
 
 ### sql_rev
 
+sql review
+
 ### required_rev
 
-### 2020_02_07_00
+### date 
+
+sql revisions date
+
+### 2021_06_17_00
+
+the date of the last update to the auth database
 
 ### Example
 
-| sql_rev             | required_rev | 2020_02_07_00 |
-|---------------------|--------------|---------------|
-| 1554142988374631100 |              |               |
-| 1579213352894781043 |              |               |
+| sql_rev             | required_rev | date          | 2021_06_17_00 |
+|---------------------|--------------|---------------|---------------|
+| 1554142988374631100 |              |               |               |
+| 1579213352894781043 |              |               |               |
+| 1620146306002634000 |              | 2021_06_17_00 |               |
```

Comment:
Remove this, unnecessary and doesn't serve any point.

Reply:
fix https://github.com/azerothcore/wiki/pull/573/commits/fed2f3ec6198906fc7275f4d0f5bcd556a354d23